src/bin/nydus-image/builder/diff.rs

```diff
@@ -792,7 +797,7 @@ impl DiffBuilder {
         for (snapshot_idx, worker) in workers.into_iter().enumerate() {
             // FIXME: the behavior of build with diff is not working.
```

Comment:
Is this still the case?

Reply:
The diff build includes two methods，`build_with_hint` (for overlayfs snapshot) and `build_with_diff` (for non-overlayfs snapshot, for example devmapper snapshot), currently we only use `build_with_hint` in acceld + containerd + overlayfs snapshotter, the support of non-overlayfs snapshotter has not been tested in detail, so a FIXME is still needed here.